### PR TITLE
DOCSUP-3539 Отработать замечания по переводу

### PR DESCRIPTION
**Pull request by damozhaeva**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):

- Documentation (changelog entry is not required)

Detailed description / Documentation draft:

- edit the Russian version of the text
- translate the text to English

### CODE REVIEW

File: docs/ru/operations/settings/settings.md [hunk 1]
```diff
@@ -1987,7 +1985,7 @@ SELECT idx, i FROM null_in WHERE i IN (1, NULL) SETTINGS transform_null_in = 1;
 
 ## output_format_pretty_grid_charset {#output-format-pretty-grid-charset}
 
-Позволяет изменить кодировку, которая используется для печати грид-границ. Доступны следующие кодировки: UTF-8, ASCII.
+ППозволяет изменить кодировку, которая используется для рисования таблицы при выводе результатов запросов. Доступны следующие кодировки: UTF-8, ASCII.
```
Comment (qoega) [inline docs/ru/operations/settings/settings.md:1988]:
```suggestion
Позволяет изменить кодировку, которая используется для отрисовки таблицы при выводе результатов запросов. Доступны следующие кодировки: UTF-8, ASCII.
```

File: docs/en/operations/settings/settings.md [hunk 2]
```diff
@@ -311,8 +311,12 @@ Enables or disables parsing enum values as enum ids for TSV input format.
 
 Possible values:
 
+<<<<<<< HEAD
+Enables or disables using default values if input data contain `NULL`, but the data type of the corresponding column in not `Nullable(T)` (for text input formats).
+=======
 -   0 — Enum values are parsed as values.
 -   1 — Enum values are parsed as enum IDs
+>>>>>>> 6325b15a63335e2efd7de1ae92d2907493a07a9c
```
Comment (qoega) [inline docs/en/operations/settings/settings.md:319]: Not ready to merge

File: docs/en/operations/settings/settings.md [hunk 3]
```diff
@@ -1314,7 +1318,15 @@ See also:
...
 
+<<<<<<< HEAD
+<<<<<<< HEAD
+Write to a quorum timeout in seconds. If the timeout has passed and no write has taken place yet, ClickHouse will generate an exception and the client must repeat the query to write the same block to the same or any other replica.
+=======
+Write to a quorum timeout in milliseconds. If the timeout has passed and no write has taken place yet, ClickHouse will generate an exception and the client must repeat the query to write the same block to the same or any other replica.
+>>>>>>> 6325b15a63335e2efd7de1ae92d2907493a07a9c
+=======
 Write to a quorum timeout in milliseconds. If the timeout has passed and no write has taken place yet, ClickHouse will generate an exception and the client must repeat the query to write the same block to the same or any other replica.
+>>>>>>> 547db452d63ba42b88e82cbe9a2aa1f5c683403f
```
Comment (qoega) [inline docs/en/operations/settings/settings.md:1329]: Not ready to merge

File: docs/en/operations/settings/settings.md [hunk 4]
```diff
@@ -1613,7 +1625,11 @@ Default value: 0.
 -   Type: seconds
 -   Default value: 60 seconds
 
+<<<<<<< HEAD
+Controls how fast errors in distributed tables are zeroed. If a replica is unavailable for some time, accumulates 5 errors, and distributed\_replica\_error\_half\_life is set to 1 second, then the replica is considered normal 3 seconds after the last error.
+=======
 Controls how fast errors in distributed tables are zeroed. If a replica is unavailable for some time, accumulates 5 errors, and distributed_replica_error_half_life is set to 1 second, then the replica is considered normal 3 seconds after the last error.
+>>>>>>> 6325b15a63335e2efd7de1ae92d2907493a07a9c
```
Comment (qoega) [inline docs/en/operations/settings/settings.md:1632]: Not ready to merge

File: docs/en/operations/settings/settings.md [hunk 5]
```diff
@@ -1859,6 +1875,8 @@ Default value: `0`.
 -   [Distributed Table Engine](../../engines/table-engines/special/distributed.md#distributed)
 -   [Managing Distributed Tables](../../sql-reference/statements/system.md#query-language-system-distributed)
 
+<<<<<<< HEAD
+=======
```
Comment (qoega) [inline docs/en/operations/settings/settings.md:1879]: Not ready to merge

File: docs/en/operations/settings/settings.md [hunk 6]
```diff
@@ -1889,6 +1907,7 @@ Default value: `1`.
 
 ## background_buffer_flush_schedule_pool_size {#background_buffer_flush_schedule_pool_size}
 
+>>>>>>> 6325b15a63335e2efd7de1ae92d2907493a07a9c
```
Comment (qoega) [inline docs/en/operations/settings/settings.md:1910]: Not ready to merge

File: docs/en/operations/settings/settings.md [hunk 7]
```diff
@@ -2205,7 +2224,11 @@ Default value: `0`.
 
 ## lock_acquire_timeout {#lock_acquire_timeout}
 
+<<<<<<< HEAD
+Defines how many seconds a locking request waits before failing. 
+=======
 Defines how many seconds a locking request waits before failing.
+>>>>>>> 6325b15a63335e2efd7de1ae92d2907493a07a9c
```
Comment (qoega) [inline docs/en/operations/settings/settings.md:2231]: Not ready to merge

**Review by qoega — CHANGES_REQUESTED** [review]

**Review by nikitamikhaylov — APPROVED** [review]

**Pull request merged**